.travis.yml

```diff
@@ -8,22 +8,16 @@ cache:
 
 matrix:
   include:
-    - php: 5.3
-      env: DEPENDENCIES='low'
-    - php: 5.3
-    - php: 5.4
-    - php: 5.5
-    - php: 5.6
     - php: 7.0
+    - php: 7.1
     - php: hhvm
 
 before_install:
   - composer self-update
 
 install:
   - export COMPOSER_ROOT_VERSION=dev-master
-  - if [ "$DEPENDENCIES" != "low" ]; then composer update; fi;
-  - if [ "$DEPENDENCIES" == "low" ]; then composer update --prefer-lowest; fi;
```

Comment:
I don't like removing low dependencies tests, please bring back `DEPENDENCIES='low'` for php 7.0